tests/README.md

```diff
@@ -21,154 +20,174 @@ You can read more about installing and configuring Codeception from the followin
+- iconv
+- igbinary
+- json
+- memcached
+- mbstring
+- mongodb
+- opcache
+- phar
+- pdo
+- pdo_mysql
+- pdo_pgsql
+- pdo_sqlite
+- redis
+- session
+- simplexml
+- tokenizer
+- yaml
+- zephir_parser
+- xdebug
+- xml
+- xmlwriter
+- zip
+- zlib
+
+## Start the environment
+We will need a terminal window, so open one if you don't have one already and navigate to the folder where you have cloned the repository (or a fork of it)
+
+Nanobox reads its configuration, so as to start the environment, from a file called `boxfile.yml` located at the root of your folder. By default the file is not there to allow for more flexibility. We have two setup files, one for PHP 7.2 and one for PHP 7.3. If you wish to set up an environment for Phalcon using PHP 7.2, you can copy the relevant file at the root of your folder:
+
+```bash
+cp -v ./tests/_ci/nanobox/boxfile.7.2.yml ./boxfile.yml
 ```
 
-*PostgreSQL*
-```sh
-psql -c 'create database phalcon_test;' -U postgres
-psql -U postgres phalcon_test -q -f tests/_data/schemas/phalcon-schema-postgresql.sql
-```
-
-**Note:** For these MySQL-related we use the user `root` without a password.
-You may need to change this in `codeception.yml` file.
-
-Obviously, Beanstalk-tests use Beanstalk, Memcached-tests use Memcached, etc.
-
-We use the following settings of these services:
+You cal also create a 7.3 environment by copying the relevant file.
 
-**Beanstalk**
-
-* Host: `127.0.0.1`
-* Port: `11300`
-
-**Memcached**
-
-* Host: `127.0.0.1`
-* Port: `11211`
-
-**SQLite**
+Run
+```bash
+nanobox run
+```
 
-* DB Name: `tests/_output/tests/phalcon_test.sqlite`
+The process will take a while (for the first time) and once it is done you will be inside the environment. The prompt of your terminal will be:
+
+```bash
+Preparing environment :
+
+....
+
+                                   **
+                                ********
+                             ***************
+                          *********************
+                            *****************
+                          ::    *********    ::
+                             ::    ***    ::
+                           ++   :::   :::   ++
+                              ++   :::   ++
+                                 ++   ++
+                                    +
+                    _  _ ____ _  _ ____ ___  ____ _  _
+                    |\ | |__| |\ | |  | |__) |  |  \/
+                    | \| |  | | \| |__| |__) |__| _/\_
+
+--------------------------------------------------------------------------------
++ You are in a Linux container
++ Your local source code has been mounted into the container
++ Changes to your code in either the container or desktop will be mirrored
++ If you run a server, access it at >> 172.18.0.2
+--------------------------------------------------------------------------------
+
+/app $
+```
 
-**MySQL**
+First update composer:
+```sh
+# run this command from project root
+/app $ composer install --dev --prefer-source
+```
```

Comment:
This is done automatically via `nanobox build`, which the first `nanobox run` will always call...